pkg/cmd/pipeline.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2019 The Knative Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	namespace string
+)
+
+// pipelinesCmd represents the pipelines command
+var pipelinesCmd = &cobra.Command{
+	Use:     "pipelines",
+	Aliases: []string{"p", "pipeline"},
+
+	Short: "Handles pipelines",
+	Long:  `add long description`, // TODO(sthaha)
+}
+
+func init() {
+	rootCmd.AddCommand(pipelinesCmd)
+
+	pipelinesCmd.PersistentFlags().StringVarP(
```

Comment:
- Shouldn't the flag be on the root command even ?
- We should have a default namespace value (can be done in a follow-up for sure).

Reply:
yes, my thoughts on this is to wait a bit until we are sure all commands need `-n ` but that may not be case already with `help` and `completion`

I agree we should find a default namespace based on the context but like you said as a separate patch.